addons/hr_holidays/models/hr_leave.py

```diff
@@ -26,6 +26,12 @@
 # See _compute_date_from_to
 DummyAttendance = namedtuple('DummyAttendance', 'hour_from, hour_to, dayofweek, day_period, week_type')
 
+def get_employee_from_context(values, context, user_employee_id):
```

Comment:
I'd pass the `env` here instead of the `context` + the `employee_id`, what do you think? :-)
IMO it makes the signature simpler

```suggestion
def get_employee_from_context(values, env):
```